docs/resources/aviatrix_account.md

```diff
@@ -255,6 +268,7 @@ The following arguments are supported:
 ### Misc.
 ~> **NOTE:** On Terraform versions 0.12.x, 0.13.x, and 0.14.x, Terraform will not detect any changes to the account when the account audit fail warning is given. In order to apply changes or set `audit_account = false`, please run `terraform apply -refresh=false`. 
 * `audit_account` - (Optional) Specify whether to enable the audit account feature. If this feature is enabled, terraform will give a warning if there is an issue with the account credentials. Changing `audit_account` to "false" will not prevent the Controller from performing account audits. It will only prevent Terraform from displaying a warning. Valid values: true, false. Default: false. Available as of provider version 2.19+. **Note: The warning may still appear for a few hours after fixing the underlying issue.**
+* `group_names` - (Optional) A list of RBAC groups name. This attribute not allow update in aviatrix_account resource, if update will show no change. Only allow to add the RBAC group which under the current user account
```

Comment:
The grammar in this should be better. Something like:
`A list of RBAC group names. This attribute should only be used when creating an account. Updating this attribute will have no effect.`